deploy/crds/workspace.che.eclipse.org_workspaceroutings_crd.yaml

```diff
@@ -115,6 +115,7 @@ spec:
               description: Workspace Routing status
               type: string
           type: object
+      type: object
```

Comment:
Please double-check if we should add this now
https://github.com/che-incubator/che-workspace-operator/commit/4992687247fd983df1a3253c8a3fe2105897f20a
https://github.com/eclipse/che-operator/pull/140

Reply:
I agree that's it's a painful one. As long as we plan to target OpenShift 3.11, and at the same time want to get the newest `operator-sdk` releases, we would have to fix this OS 3.11 incompatibility ourselves.
Maybe a bit like what has been done in the `che-operator` recently: https://github.com/eclipse/che-operator/pull/180/files#diff-8a85834afbbc2e0c3afa67168071e391R34